bin/trim-shrinkwrap.js

```diff
@@ -9,6 +9,19 @@
 const fs = require( 'fs' );
 const shrinkwrap = require( '../npm-shrinkwrap.json' );
 
+( function removeOptionalDeps( root ) {
+	if ( ! root.dependencies ) {
+		return;
+	}
+	Object.keys( root.dependencies ).forEach( function ( dep ) {
+		if ( root.dependencies[ dep ].optional ) {
+			delete root.dependencies[ dep ];
+		} else {
+			removeOptionalDeps( root.dependencies[ dep ] );
+		}
+	} );
+} )( shrinkwrap );
+
```

Comment:
I'm consistently running into problems when I do `update-deps` then `build-docker`:

```
npm ERR! code EBADPLATFORM
npm ERR! notsup Unsupported platform for fsevents@1.1.1: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm ERR! notsup Valid OS:    darwin
npm ERR! notsup Valid Arch:  any
npm ERR! notsup Actual OS:   linux
npm ERR! notsup Actual Arch: x64
```

When I remove this, I don't have the problem. I'm going to revert it in another commit @DanReyLop